WordPress/Classes/ViewRelated/Reader/ReaderShowMenuAction.swift

```diff
@@ -132,6 +132,24 @@ final class ReaderShowMenuAction {
                                             ReaderShareAction().execute(with: post, context: context, anchor: anchor, vc: vc)
         })
 
+        // Comment Subscription (Follow Comments by Email)
+        if post.canSubscribeComments {
+            let buttonTitle = post.isSubscribedComments ? ReaderPostMenuButtonTitles.unFollowConversation : ReaderPostMenuButtonTitles.followConversation
+            alertController.addActionWithTitle(buttonTitle,
+                                               style: .default,
+                                               handler: { (action: UIAlertAction) in
+                                                if let post: ReaderPost = ReaderActionHelpers.existingObject(for: post.objectID, in: context) {
+                                                    ReaderSubscribeCommentsAction().execute(with: post,
+                                                                                 context: context,
+                                                                                 completion: {
+                                                                                    // No op
+                                                                                 }, failure: { _ in
+                                                                                    // No op.
+                                                                                 })
```

Comment:
Since these blocks are optional, can they just be removed?

Reply:
Oh good catch. Yeah they totally can.